Formal2024/Workshops/workshop3.lean

```diff
@@ -0,0 +1,210 @@
+/-
+  # Logic: Conjunction, Iff, and Disjunction
+  References: [MIL] Mathematics in Lean, [Tut] Tutorials project.
+  Most of the demonstration section comes [MIL].
+-/
+
+import Mathlib.Tactic
+
+/-!
+  # Conjunction
+-/
+
+-- The symbol for conjunction is `∧`, which you can write as `\and`.
+
+-- To prove a conjunction, we use the `constructor` tactic.
+-- This will split the goal into two, one for each part of the conjunction.
+-- (For people familiar with Lean 3: the `split` tactic is gone in Lean 4.)
+
+example (P Q : Prop) (hp : P) (hq : Q) : P ∧ Q := by
+  constructor
+  · exact hp
```

Comment:
We've seen them before, but maybe it's worth saying something about the \. to handle multiple cases?

Reply:
I'll mention it during the demo, but since we've already explained it in a previous file I think it's fine a s is.